src/egl/drivers/dri2/platform_android_gralloc.cpp

```diff
@@ -0,0 +1,23 @@
+#include "egl_dri2.h"
+#include <cutils/log.h>
+extern "C" int resolve_gralloc1_functions(struct dri2_egl_display *dri2_dpy);
+
+int resolve_gralloc1_functions(struct dri2_egl_display *dri2_dpy){
```

Comment:
I don't see anything requiring C++ here but OK we can let it be like this for now. If upstreaming wants C, then we can change it later.